cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2465,20 +2490,35 @@ macro(build_grpc)
   string(REPLACE ";" "|" GRPC_PREFIX_PATH_ALT_SEP "${GRPC_CMAKE_PREFIX}")
 
   set(GRPC_CMAKE_ARGS
+      -DBUILD_SHARED_LIBS=OFF
       -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+      -DCMAKE_CXX_FLAGS=${GRPC_CMAKE_CXX_FLAGS}
```

Comment:
Can we use `EP_COMMON_CMAKE_ARGS` for `GRPC_CMAKE_ARGS`?

It seems that https://github.com/grpc/grpc/issues/13856 has been fixed in gRPC 1.33.1 or later.